packages/grafana-ui/src/Intro.story.mdx

```diff
@@ -1,12 +1,26 @@
 <Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false }, id: 1 }} />
 
-# Grafana UI library
+# Grafana design system
 
 > **@grafana/ui is currently in BETA**.
 
-@grafana/ui is a collection of components used by [Grafana](https://github.com/grafana/grafana)
+With the design system @grafana/ui, we want to democratize development. This library of reusable [Grafana](https://github.com/grafana/grafana) components and guidelines helps you with contribution and plugin development.
 
-If you're a plugin developer, you can use these component to give your plugin an official look-and-feel.
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
```

Comment:
I think this paragraph basically repeats the previous one and maybe could be removed.